### ISSUE TITLE
Allow user to smooth bounding box transition

### ISSUE THREAD

**Issue by balintlaczko** [fourMs/VideoAnalysis#32]
Even a little smoothing of what `[jit.findbounds]` returns can get rid of a lot of flickering.

**Comment by alexarje** [comment]
Yes, good point.

**Comment by aleksati** [comment]
I added [VA_findbounds_interpolation] to the box-image now which interpolates between median values coming from the [jit.findbounds]. This should smooth the drawing of the box rectangle pretty well. These median values are based on N-large lists of boundmin and boundmax values.

Therefore, I added a sensitivity parameter to the box-image section in the UI that specifies how large the lists are in which the median values are derived. This gets rid of a lot of flickering especially when tuning the sensitivity together with the radius and threshold parameters. 

However, it might need some more finetuning. Especially to find a good default setting. Great if we can test this a bit further.

**Comment by alexarje** [comment]
Yes, I am struggling to find good values here. This function is not very important, I think. I implemented it for a particular project, but I haven't really used it much myself, and not sure if others have either. So I wouldn't say that it is high priority.

**Comment by aleksati** [comment]
Jepp, there is still some work to be done here, and I have a few things in mind. But let's not focus on this right now, as you said. Ill relabel this with a 3.0 label.
